src/drive/web/modules/actions/useActions.spec.jsx

```diff
@@ -0,0 +1,422 @@
+import React from 'react'
+import { renderHook } from '@testing-library/react-hooks'
+import { createMockClient } from 'cozy-client'
+
+import { ModalContext } from 'drive/lib/ModalContext'
+import { SharingContext, ShareModal } from 'cozy-sharing'
+import { RouterContext } from 'drive/lib/RouterContext'
+import AppLike from '../../../../../test/components/AppLike'
+import DeleteConfirm from 'drive/web/modules/drive/DeleteConfirm'
+import MoveModal from 'drive/web/modules/move/MoveModal'
+import { EditDocumentQualification } from 'cozy-scanner'
+import { createStore } from 'redux'
+import { exportFilesNative, downloadFiles, openFileWith } from './utils'
+import * as rename from 'drive/web/modules/drive/rename'
+
+import useActions from './useActions'
+
+jest.mock('./utils', () => ({
+  isAnyFileReferencedByAlbum: jest.fn(),
+  exportFilesNative: jest.fn(),
+  downloadFiles: jest.fn(),
+  trashFiles: jest.fn(),
+  openFileWith: jest.fn()
+}))
+
+describe('useActions', () => {
+  const mockFolderId = '123'
+  const mockStore = createStore(() => ({
+    mobile: {
+      url: 'cozy-url://'
+    }
+  }))
+  const mockModalContextValue = {
+    pushModal: jest.fn()
+  }
+  const mockSharingContextValue = {
+    hasWriteAccess: jest.fn(),
+    refresh: jest.fn()
+  }
+  const mockRouterContextValue = {
+    router: {
+      push: jest.fn()
+    },
+    location: { pathname: '/folder' }
+  }
+  const mockClient = createMockClient({})
+
+  beforeEach(() => {
+    jest.resetAllMocks()
+    global.__TARGET__ = 'desktop'
```

Comment:
Is it a real target ? I thought __TARGET__ could only be "browser" or "mobile". Recently I had to add it to the globals exposed by jest. I put "browser" so I think the above line is not needed. I think we should explicitly set it when setting it to "mobile".

Reply:
Ah you're correct, it's `browser` and not `desktop`. The new global isn't merged yet, it's in #2032, but I can use the same approach 👍